lib/async.js

```diff
@@ -36,6 +36,18 @@ var maybeUnwrapSymlink = function maybeUnwrapSymlink(x, opts, cb) {
     }
 };
 
+var getPackageCandidates = function getPackageCandidates(x, start, opts) {
+    var dirs = nodeModulesPaths(start, opts, x);
+    for (var i = 0; i < dirs.length; i++) {
+        dirs[i] = path.join(dirs[i], x);
+    }
+    return dirs;
+};
+
+var defaultPackageIterator = function defaultPackageIterator(x, start, getPackageCandidates, opts) {
+    return getPackageCandidates();
```

Comment:
Nit: this is really confusing. I saw a function with the same name defined right before this function, which takes 3 arguments. It took me a while to realize that this isn't calling that function, but the one passed as a parameter.

Maybe the one on line 39 could be renamed to `getPackageNodeModulesCandidates`?

Reply:
```suggestion
    return defaultGetPackageCandidates(x, start, opts);
```